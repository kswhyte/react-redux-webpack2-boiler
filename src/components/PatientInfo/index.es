import Component from './PatientInfo';
export default Component;
