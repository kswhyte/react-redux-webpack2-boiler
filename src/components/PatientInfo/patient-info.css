@import "imports/_helpers.css";
@import "imports/_layout.css";
@import "imports/_form-elements.css";
@import "imports/_typography.css";

.patient-info-wrapper {
  & button {
    &.next-btn {
      float: right;
    }
  }

  & row {
    margin: 0 -5px;
  }


}
