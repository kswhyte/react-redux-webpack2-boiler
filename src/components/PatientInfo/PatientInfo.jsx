/*eslint no-unused-vars: */

import React, { Component } from 'react';
import { PropTypes } from 'prop-types';
// import isEmail from '../../helpers/isEmail';

import './patient-info.css';

const propTypes = {
  children: PropTypes.object,
  generalInfo: PropTypes.object.required
};

class PatientInfo extends Component {
  constructor(props) {
    super(props);
    this.state = {
      firstName: '',
      lastName: '',
      gender: '',
      emailAddress: '',
      dobMonth: '',
      dobDay: '',
      dobYear: '',
      phoneNumber: '',
      phoneType: '',
      contactMethod: '',
      street: '',
      city: '',
      state: '',
      zip: '',
      insuranceCarrierPrimary: '',
      planTypePrimary: '',
      planNumberPrimary: '',
      insuranceCarrierAlternate: '',
      planTypeAlternate: '',
      planNumberAlternate: '',
      referringPhysician: ''
    };
  }

  componentDidMount() {}

  render() {
    console.log('----state', this.state);
    console.log('----props', this.props);
    return (
      <div className="patient-info-wrapper">
        <div className="tab-content clearfix">
          <div className="general-info-form tab-pane active">
            <form>
              {/* Row1 */}
              <div className="row sm-spacer">
                <div className="col-sm-3 no-margin">
                  <label>PATIENT FIRST NAME &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    value={this.props.generalInfo.firstName ? this.props.generalInfo.firstName : this.state.firstName}
                    placeholder="Patient First Name"
                    onChange={e => this.setState({ firstName: e.target.value })}
                    required
                  />
                </div>

                <div className="col-sm-3 no-margin">
                  <label>PATIENT LAST NAME &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    value={this.props.generalInfo.lastName ? this.props.generalInfo.lastName : this.state.lastName}
                    placeholder="Patient Last Name"
                    onChange={e => this.setState({ lastName: e.target.value })}
                    required
                  />
                </div>

                <div className="col-sm-2 no-margin">
                  <label>GENDER &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ gender: e.target.value })}
                    required
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-4 no-margin">
                  <label>EMAIL ADDRESS &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    value={
                      this.props.generalInfo.emailAddress
                        ? this.props.generalInfo.emailAddress
                        : this.state.emailAddress
                    }
                    placeholder="Email Address"
                    onChange={e => this.setState({ emailAddress: e.target.value })}
                    required
                  />
                </div>
              </div>

              {/* Row2 */}
              <div className="row sm-spacer">
                <div className="col-sm-3 no-margin">
                  <label>PATIENT DATE OF BIRTH &#42;</label>
                  <select className="select-dropdown" onChange={e => this.setState({ dobMonth: e.target.value })}>
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-1 no-margin">
                  <label />
                  <select
                    className="select-dropdown dropdown-affiliate"
                    onChange={e => this.setState({ dobDay: e.target.value })}
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label />
                  <select
                    className="select-dropdown dropdown-affiliate"
                    onChange={e => this.setState({ dobYear: e.target.value })}
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label>PHONE NUMBER &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    value={
                      this.props.generalInfo.homePhoneNumber
                        ? this.props.generalInfo.homePhoneNumber
                        : this.state.phoneNumber
                    }
                    placeholder="Phone Number"
                    onChange={e => this.setState({ phoneNumber: e.target.value })}
                    required
                  />
                </div>

                <div className="col-sm-2 no-margin">
                  <label>PHONE TYPE &#42;</label>
                  <select className="select-dropdown" onChange={e => this.setState({ phoneType: e.target.value })}>
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label>CONTACT METHOD &#42;</label>
                  <select className="select-dropdown" onChange={e => this.setState({ contactMethod: e.target.value })}>
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>
              </div>

              {/* Row3 */}
              <div className="row sm-spacer">
                <div className="col-sm-4 no-margin">
                  <label>STREET &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    onChange={e => this.setState({ street: e.target.value })}
                    required
                  />
                </div>

                <div className="col-sm-3 no-margin">
                  <label>CITY STATE ZIP &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    placeholder="City"
                    onChange={e => this.setState({ city: e.target.value })}
                    required
                  />
                </div>
                <div className="col-sm-2 no-margin">
                  <label />
                  <select
                    className="select-dropdown dropdown-affiliate"
                    onChange={e => this.setState({ state: e.target.value })}
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label />
                  <input
                    className="form-group"
                    type="text"
                    placeholder="Zip"
                    onChange={e => this.setState({ zipCode: e.target.value })}
                    required
                  />
                </div>
              </div>

              {/* Row4 */}
              <div className="row sm-spacer no-margin">
                <div className="col-sm-3">
                  <label>PRIMARY INSURANCE CARRIER &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ insuranceCarrierPrimary: e.target.value })}
                    required
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-1 no-margin">
                  <label>PLAN TYPE &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ planTypePrimary: e.target.value })}
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label>PLAN NO. &#42;</label>
                  <input placeholder="Plan Number" className="form-group" type="text" required />
                </div>
              </div>

              {/* Row5 */}
              <div className="row sm-spacer no-margin">
                <div className="col-sm-3">
                  <label>ALTERNATE INSURANCE CARRIER &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ insuranceCarrierAlternate: e.target.value })}
                    required
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>

                <div className="col-sm-1 no-margin">
                  <label>PLAN TYPE &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ planTypeAlternate: e.target.value })}
                  >
                    <option value="ppo" selected>
                      PPO
                    </option>
                  </select>
                </div>

                <div className="col-sm-2 no-margin">
                  <label>PLAN NO. &#42;</label>
                  <input
                    className="form-group"
                    type="text"
                    placeholder="Plan Number"
                    onChange={e => this.setState({ planTypeAlternate: e.target.value })}
                    required
                  />
                </div>
              </div>

              {/* Row6 */}
              <div className="row sm-spacer">
                <div className="col-sm-4">
                  <label>REFERRING PHYSICIAN &#42;</label>
                  <select
                    className="select-dropdown"
                    onChange={e => this.setState({ referringPhysician: e.target.value })}
                  >
                    <option value="select" selected>
                      Select
                    </option>
                  </select>
                </div>
              </div>

              <div className="row sm-spacer">
                <div className="footer-line" />
                <div className="break-line footer-line" />
              </div>

              {/* Buttons */}
              <div className="row sm-spacer button-wrapper">
                <div className="col-sm-6">
                  <button className="back-btn" tabIndex={1}>
                    BACK
                  </button>
                </div>

                <div className="col-sm-6">
                  <button className="next-btn" type="submit" tabIndex={1}>
                    NEXT
                  </button>
                </div>
              </div>
            </form>
          </div>
        </div>
      </div>
    );
  }
}

PatientInfo.propTypes = propTypes;

export default PatientInfo;
