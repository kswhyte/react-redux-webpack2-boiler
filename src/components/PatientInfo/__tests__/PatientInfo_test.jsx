import React from 'react';
// import { storiesOf, action, linkTo } from '@kadira/storybook';
import PatientInfo from '../PatientInfo';

// import { specs, describe, it } from 'storybook-addon-specifications';

import { storiesOf, describe, it, specs } from '../../../../.storybook/facade';

import { shallow } from 'enzyme';
import expect from 'expect';

const stories = storiesOf('PatientInfo', module);

stories.add('Patient Info Story', () => {
  const props = {
    generalInfo : {firstName: "test"}
  }
  const patientInfoStory = <PatientInfo {...props} />;

  specs(() =>
    describe('Show a successful alert', () => {
      it('Should render the PatientInfo component without crashing', () => {
        let output = shallow(patientInfoStory);
        expect(output.find('.patient-info-wrapper').length).toEqual(1);
      });
    })
  );
  return patientInfoStory;
});
