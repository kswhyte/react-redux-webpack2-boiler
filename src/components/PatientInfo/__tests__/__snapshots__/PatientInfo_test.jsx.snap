// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Patient Info Story 1`] = `
<div
  className="patient-info-wrapper"
>
  <div
    className="tab-content clearfix"
  >
    <div
      className="general-info-form tab-pane active"
    >
      <form
        onSubmit={[Function]}
      >
        <div
          className="row xs-spacer"
        >
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              PATIENT FIRST NAME *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="Patient First Name"
              required={true}
              type="text"
              value=""
            />
          </div>
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              PATIENT LAST NAME *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="Patient Last Name"
              required={true}
              type="text"
              value=""
            />
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              GENDER *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
              required={true}
            >
              <option
                label="Female"
                value="F"
              >
                Female
              </option>
              <option
                label="Male"
                value="M"
              >
                Male
              </option>
            </select>
          </div>
          <div
            className="col-sm-4 no-margin padding-10"
          >
            <label>
              EMAIL ADDRESS *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="Email Address"
              required={true}
              type="email"
              value=""
            />
          </div>
        </div>
        <div
          className="row xs-spacer"
        >
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              PATIENT DATE OF BIRTH *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
            >
              <option
                selected={true}
                value="select"
              >
                Select
              </option>
            </select>
          </div>
          <div
            className="col-sm-1 no-margin padding-10"
          >
            <label />
            <select
              className="select-dropdown dropdown-affiliate form-control"
              onChange={[Function]}
              value=""
            >
              <option
                label={1}
                value={1}
              >
                1
              </option>
              <option
                label={2}
                value={2}
              >
                2
              </option>
              <option
                label={3}
                value={3}
              >
                3
              </option>
              <option
                label={4}
                value={4}
              >
                4
              </option>
              <option
                label={5}
                value={5}
              >
                5
              </option>
              <option
                label={6}
                value={6}
              >
                6
              </option>
              <option
                label={7}
                value={7}
              >
                7
              </option>
              <option
                label={8}
                value={8}
              >
                8
              </option>
              <option
                label={9}
                value={9}
              >
                9
              </option>
              <option
                label={10}
                value={10}
              >
                10
              </option>
              <option
                label={11}
                value={11}
              >
                11
              </option>
              <option
                label={12}
                value={12}
              >
                12
              </option>
              <option
                label={13}
                value={13}
              >
                13
              </option>
              <option
                label={14}
                value={14}
              >
                14
              </option>
              <option
                label={15}
                value={15}
              >
                15
              </option>
              <option
                label={16}
                value={16}
              >
                16
              </option>
              <option
                label={17}
                value={17}
              >
                17
              </option>
              <option
                label={18}
                value={18}
              >
                18
              </option>
              <option
                label={19}
                value={19}
              >
                19
              </option>
              <option
                label={20}
                value={20}
              >
                20
              </option>
              <option
                label={21}
                value={21}
              >
                21
              </option>
              <option
                label={22}
                value={22}
              >
                22
              </option>
              <option
                label={23}
                value={23}
              >
                23
              </option>
              <option
                label={24}
                value={24}
              >
                24
              </option>
              <option
                label={25}
                value={25}
              >
                25
              </option>
              <option
                label={26}
                value={26}
              >
                26
              </option>
              <option
                label={27}
                value={27}
              >
                27
              </option>
              <option
                label={28}
                value={28}
              >
                28
              </option>
              <option
                label={29}
                value={29}
              >
                29
              </option>
              <option
                label={30}
                value={30}
              >
                30
              </option>
              <option
                label={31}
                value={31}
              >
                31
              </option>
            </select>
          </div>
          <div
            className="col-sm-1 no-margin padding-10"
          >
            <label />
            <select
              className="select-dropdown dropdown-affiliate form-control"
              onChange={[Function]}
              value=""
            >
              <option
                label={2017}
                value={2017}
              >
                2017
              </option>
              <option
                label={2016}
                value={2016}
              >
                2016
              </option>
              <option
                label={2015}
                value={2015}
              >
                2015
              </option>
              <option
                label={2014}
                value={2014}
              >
                2014
              </option>
              <option
                label={2013}
                value={2013}
              >
                2013
              </option>
              <option
                label={2012}
                value={2012}
              >
                2012
              </option>
              <option
                label={2011}
                value={2011}
              >
                2011
              </option>
              <option
                label={2010}
                value={2010}
              >
                2010
              </option>
              <option
                label={2009}
                value={2009}
              >
                2009
              </option>
              <option
                label={2008}
                value={2008}
              >
                2008
              </option>
              <option
                label={2007}
                value={2007}
              >
                2007
              </option>
              <option
                label={2006}
                value={2006}
              >
                2006
              </option>
              <option
                label={2005}
                value={2005}
              >
                2005
              </option>
              <option
                label={2004}
                value={2004}
              >
                2004
              </option>
              <option
                label={2003}
                value={2003}
              >
                2003
              </option>
              <option
                label={2002}
                value={2002}
              >
                2002
              </option>
              <option
                label={2001}
                value={2001}
              >
                2001
              </option>
              <option
                label={2000}
                value={2000}
              >
                2000
              </option>
              <option
                label={1999}
                value={1999}
              >
                1999
              </option>
              <option
                label={1998}
                value={1998}
              >
                1998
              </option>
              <option
                label={1997}
                value={1997}
              >
                1997
              </option>
              <option
                label={1996}
                value={1996}
              >
                1996
              </option>
              <option
                label={1995}
                value={1995}
              >
                1995
              </option>
              <option
                label={1994}
                value={1994}
              >
                1994
              </option>
              <option
                label={1993}
                value={1993}
              >
                1993
              </option>
              <option
                label={1992}
                value={1992}
              >
                1992
              </option>
              <option
                label={1991}
                value={1991}
              >
                1991
              </option>
              <option
                label={1990}
                value={1990}
              >
                1990
              </option>
              <option
                label={1989}
                value={1989}
              >
                1989
              </option>
              <option
                label={1988}
                value={1988}
              >
                1988
              </option>
              <option
                label={1987}
                value={1987}
              >
                1987
              </option>
              <option
                label={1986}
                value={1986}
              >
                1986
              </option>
              <option
                label={1985}
                value={1985}
              >
                1985
              </option>
              <option
                label={1984}
                value={1984}
              >
                1984
              </option>
              <option
                label={1983}
                value={1983}
              >
                1983
              </option>
              <option
                label={1982}
                value={1982}
              >
                1982
              </option>
              <option
                label={1981}
                value={1981}
              >
                1981
              </option>
              <option
                label={1980}
                value={1980}
              >
                1980
              </option>
              <option
                label={1979}
                value={1979}
              >
                1979
              </option>
              <option
                label={1978}
                value={1978}
              >
                1978
              </option>
              <option
                label={1977}
                value={1977}
              >
                1977
              </option>
              <option
                label={1976}
                value={1976}
              >
                1976
              </option>
              <option
                label={1975}
                value={1975}
              >
                1975
              </option>
              <option
                label={1974}
                value={1974}
              >
                1974
              </option>
              <option
                label={1973}
                value={1973}
              >
                1973
              </option>
              <option
                label={1972}
                value={1972}
              >
                1972
              </option>
              <option
                label={1971}
                value={1971}
              >
                1971
              </option>
              <option
                label={1970}
                value={1970}
              >
                1970
              </option>
              <option
                label={1969}
                value={1969}
              >
                1969
              </option>
              <option
                label={1968}
                value={1968}
              >
                1968
              </option>
              <option
                label={1967}
                value={1967}
              >
                1967
              </option>
              <option
                label={1966}
                value={1966}
              >
                1966
              </option>
              <option
                label={1965}
                value={1965}
              >
                1965
              </option>
              <option
                label={1964}
                value={1964}
              >
                1964
              </option>
              <option
                label={1963}
                value={1963}
              >
                1963
              </option>
              <option
                label={1962}
                value={1962}
              >
                1962
              </option>
              <option
                label={1961}
                value={1961}
              >
                1961
              </option>
              <option
                label={1960}
                value={1960}
              >
                1960
              </option>
              <option
                label={1959}
                value={1959}
              >
                1959
              </option>
              <option
                label={1958}
                value={1958}
              >
                1958
              </option>
              <option
                label={1957}
                value={1957}
              >
                1957
              </option>
              <option
                label={1956}
                value={1956}
              >
                1956
              </option>
              <option
                label={1955}
                value={1955}
              >
                1955
              </option>
              <option
                label={1954}
                value={1954}
              >
                1954
              </option>
              <option
                label={1953}
                value={1953}
              >
                1953
              </option>
              <option
                label={1952}
                value={1952}
              >
                1952
              </option>
              <option
                label={1951}
                value={1951}
              >
                1951
              </option>
              <option
                label={1950}
                value={1950}
              >
                1950
              </option>
              <option
                label={1949}
                value={1949}
              >
                1949
              </option>
              <option
                label={1948}
                value={1948}
              >
                1948
              </option>
              <option
                label={1947}
                value={1947}
              >
                1947
              </option>
              <option
                label={1946}
                value={1946}
              >
                1946
              </option>
              <option
                label={1945}
                value={1945}
              >
                1945
              </option>
              <option
                label={1944}
                value={1944}
              >
                1944
              </option>
              <option
                label={1943}
                value={1943}
              >
                1943
              </option>
              <option
                label={1942}
                value={1942}
              >
                1942
              </option>
              <option
                label={1941}
                value={1941}
              >
                1941
              </option>
              <option
                label={1940}
                value={1940}
              >
                1940
              </option>
              <option
                label={1939}
                value={1939}
              >
                1939
              </option>
              <option
                label={1938}
                value={1938}
              >
                1938
              </option>
              <option
                label={1937}
                value={1937}
              >
                1937
              </option>
              <option
                label={1936}
                value={1936}
              >
                1936
              </option>
              <option
                label={1935}
                value={1935}
              >
                1935
              </option>
              <option
                label={1934}
                value={1934}
              >
                1934
              </option>
              <option
                label={1933}
                value={1933}
              >
                1933
              </option>
              <option
                label={1932}
                value={1932}
              >
                1932
              </option>
              <option
                label={1931}
                value={1931}
              >
                1931
              </option>
              <option
                label={1930}
                value={1930}
              >
                1930
              </option>
              <option
                label={1929}
                value={1929}
              >
                1929
              </option>
              <option
                label={1928}
                value={1928}
              >
                1928
              </option>
              <option
                label={1927}
                value={1927}
              >
                1927
              </option>
              <option
                label={1926}
                value={1926}
              >
                1926
              </option>
              <option
                label={1925}
                value={1925}
              >
                1925
              </option>
              <option
                label={1924}
                value={1924}
              >
                1924
              </option>
              <option
                label={1923}
                value={1923}
              >
                1923
              </option>
              <option
                label={1922}
                value={1922}
              >
                1922
              </option>
              <option
                label={1921}
                value={1921}
              >
                1921
              </option>
              <option
                label={1920}
                value={1920}
              >
                1920
              </option>
              <option
                label={1919}
                value={1919}
              >
                1919
              </option>
              <option
                label={1918}
                value={1918}
              >
                1918
              </option>
              <option
                label={1917}
                value={1917}
              >
                1917
              </option>
              <option
                label={1916}
                value={1916}
              >
                1916
              </option>
              <option
                label={1915}
                value={1915}
              >
                1915
              </option>
              <option
                label={1914}
                value={1914}
              >
                1914
              </option>
              <option
                label={1913}
                value={1913}
              >
                1913
              </option>
              <option
                label={1912}
                value={1912}
              >
                1912
              </option>
              <option
                label={1911}
                value={1911}
              >
                1911
              </option>
              <option
                label={1910}
                value={1910}
              >
                1910
              </option>
            </select>
          </div>
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              PHONE NUMBER *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              pattern="[0-9]{10}"
              placeholder="Phone Number"
              required={true}
              type="text"
              value=""
            />
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              PHONE TYPE *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
              value=""
            >
              <option
                label="Mobile"
                value={0}
              >
                Mobile
              </option>
              <option
                label="Home"
                value={1}
              >
                Home
              </option>
            </select>
          </div>
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              CONTACT METHOD *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
              value=""
            >
              <option
                selected={true}
                value="select"
              >
                SMS
              </option>
            </select>
          </div>
        </div>
        <div
          className="row xs-spacer"
        >
          <div
            className="col-sm-4 no-margin padding-10"
          >
            <label>
              STREET *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              required={true}
              type="text"
              value=""
            />
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              CITY STATE ZIP *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="City"
              required={true}
              type="text"
              value=""
            />
          </div>
          <div
            className="col-sm-1 no-margin padding-10"
          >
            <label />
            <select
              className="select-dropdown dropdown-affiliate form-control"
              onChange={[Function]}
              value=""
            >
              <option
                label="Select"
                value=""
              >
                Select
              </option>
              <option
                label="Alabama"
                value="AL"
              >
                Alabama
              </option>
              <option
                label="Alaska"
                value="AK"
              >
                Alaska
              </option>
              <option
                label="Arizona"
                value="AZ"
              >
                Arizona
              </option>
              <option
                label="Arkansas"
                value="AR"
              >
                Arkansas
              </option>
              <option
                label="California"
                value="CA"
              >
                California
              </option>
              <option
                label="Colorado"
                value="CO"
              >
                Colorado
              </option>
              <option
                label="Connecticut"
                value="CT"
              >
                Connecticut
              </option>
              <option
                label="Delaware"
                value="DE"
              >
                Delaware
              </option>
              <option
                label="District Of Columbia"
                value="DC"
              >
                District Of Columbia
              </option>
              <option
                label="Florida"
                value="FL"
              >
                Florida
              </option>
              <option
                label="Georgia"
                value="GA"
              >
                Georgia
              </option>
              <option
                label="Guam"
                value="GU"
              >
                Guam
              </option>
              <option
                label="Hawaii"
                value="HI"
              >
                Hawaii
              </option>
              <option
                label="Idaho"
                value="ID"
              >
                Idaho
              </option>
              <option
                label="Illinois"
                value="IL"
              >
                Illinois
              </option>
              <option
                label="Indiana"
                value="IN"
              >
                Indiana
              </option>
              <option
                label="Iowa"
                value="IA"
              >
                Iowa
              </option>
              <option
                label="Kansas"
                value="KS"
              >
                Kansas
              </option>
              <option
                label="Kentucky"
                value="KY"
              >
                Kentucky
              </option>
              <option
                label="Louisiana"
                value="LA"
              >
                Louisiana
              </option>
              <option
                label="Maine"
                value="ME"
              >
                Maine
              </option>
              <option
                label="Maryland"
                value="MD"
              >
                Maryland
              </option>
              <option
                label="Massachusetts"
                value="MA"
              >
                Massachusetts
              </option>
              <option
                label="Michigan"
                value="MI"
              >
                Michigan
              </option>
              <option
                label="Minnesota"
                value="MN"
              >
                Minnesota
              </option>
              <option
                label="Mississippi"
                value="MS"
              >
                Mississippi
              </option>
              <option
                label="Missouri"
                value="MO"
              >
                Missouri
              </option>
              <option
                label="Montana"
                value="MT"
              >
                Montana
              </option>
              <option
                label="Nebraska"
                value="NE"
              >
                Nebraska
              </option>
              <option
                label="Nevada"
                value="NV"
              >
                Nevada
              </option>
              <option
                label="New Hampshire"
                value="NH"
              >
                New Hampshire
              </option>
              <option
                label="New Jersey"
                value="NJ"
              >
                New Jersey
              </option>
              <option
                label="New Mexico"
                value="NM"
              >
                New Mexico
              </option>
              <option
                label="New York"
                value="NY"
              >
                New York
              </option>
              <option
                label="North Carolina"
                value="NC"
              >
                North Carolina
              </option>
              <option
                label="North Dakota"
                value="ND"
              >
                North Dakota
              </option>
              <option
                label="Ohio"
                value="OH"
              >
                Ohio
              </option>
              <option
                label="Oklahoma"
                value="OK"
              >
                Oklahoma
              </option>
              <option
                label="Oregon"
                value="OR"
              >
                Oregon
              </option>
              <option
                label="Palau"
                value="PW"
              >
                Palau
              </option>
              <option
                label="Pennsylvania"
                value="PA"
              >
                Pennsylvania
              </option>
              <option
                label="Puerto Rico"
                value="PR"
              >
                Puerto Rico
              </option>
              <option
                label="Rhode Island"
                value="RI"
              >
                Rhode Island
              </option>
              <option
                label="South Carolina"
                value="SC"
              >
                South Carolina
              </option>
              <option
                label="South Dakota"
                value="SD"
              >
                South Dakota
              </option>
              <option
                label="Tennessee"
                value="TN"
              >
                Tennessee
              </option>
              <option
                label="Texas"
                value="TX"
              >
                Texas
              </option>
              <option
                label="Utah"
                value="UT"
              >
                Utah
              </option>
              <option
                label="Vermont"
                value="VT"
              >
                Vermont
              </option>
              <option
                label="Virgin Islands"
                value="VI"
              >
                Virgin Islands
              </option>
              <option
                label="Virginia"
                value="VA"
              >
                Virginia
              </option>
              <option
                label="Washington"
                value="WA"
              >
                Washington
              </option>
              <option
                label="West Virginia"
                value="WV"
              >
                West Virginia
              </option>
              <option
                label="Wisconsin"
                value="WI"
              >
                Wisconsin
              </option>
              <option
                label="Wyoming"
                value="WY"
              >
                Wyoming
              </option>
            </select>
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label />
            <input
              className="form-group input-affiliate"
              onChange={[Function]}
              pattern="[0-9]{5}"
              placeholder="Zip"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="row xs-spacer"
        >
          <div
            className="col-sm-4 padding-10"
          >
            <label>
              PRIMARY INSURANCE CARRIER *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
              value=""
            >
              <option
                selected={true}
                value="select"
              >
                default
              </option>
              required
            </select>
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              PLAN TYPE *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
            >
              <option
                selected={true}
                value="select"
              >
                dflt
              </option>
            </select>
          </div>
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              PLAN NO. *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="Plan Number"
              required={true}
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="row sm-spacer"
        >
          <div
            className="col-sm-4 padding-10"
          >
            <label>
              ALTERNATE INSURANCE CARRIER
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
            >
              <option
                selected={true}
                value="select"
              >
                Select
              </option>
            </select>
          </div>
          <div
            className="col-sm-2 no-margin padding-10"
          >
            <label>
              PLAN TYPE *
            </label>
            <select
              className="select-dropdown form-control"
              onChange={[Function]}
            >
              <option
                selected={true}
                value="ppo"
              >
                PPO
              </option>
            </select>
          </div>
          <div
            className="col-sm-3 no-margin padding-10"
          >
            <label>
              PLAN NO. *
            </label>
            <input
              className="form-group"
              onChange={[Function]}
              placeholder="Plan Number"
              type="text"
              value=""
            />
          </div>
        </div>
        <div
          className="row xs-spacer"
        >
          <div
            className="col-sm-4 padding-10"
          >
            <label>
              REFERRING PHYSICIAN *
            </label>
            <select
              className="select-dropdown"
              onChange={[Function]}
            >
              <option
                selected={true}
                value="select"
              >
                Select
              </option>
            </select>
          </div>
        </div>
        <div
          className="row sm-spacer"
        >
          <div
            className="col-sm-5"
          >
            <div />
          </div>
        </div>
        <div
          className="row sm-spacer"
        >
          <div
            className="break-line"
          />
        </div>
        <div
          className="row xs-spacer sm-spacer"
        >
          <div
            className="col-sm-6 no-padding"
          >
            <button
              className="back-btn"
              tabIndex={1}
            >
              BACK
            </button>
          </div>
          <div
            className="col-sm-6 no-padding"
          >
            <button
              className="next-btn"
              onClick={undefined}
              tabIndex={1}
              type="submit"
            >
              NEXT
            </button>
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
